operations/configure-cf-services.yml

```diff
@@ -46,29 +46,3 @@
     uris:
       - ((deployment_name))-cf-scalingengine.((system_domain))
 
-
-## EVENTGENERATOR - Enable cf Server to receive calls from api running on cf --
```

Comment:
bring back this code to allow rollback on vms when stopping bosh vm and using app instead